src/TimeZones.jl

```diff
@@ -31,11 +31,11 @@ export TimeZone, @tz_str, istimezone, FixedTimeZone, VariableTimeZone, ZonedDate
     # TZData
     build
 
-const PKG_DIR = normpath(joinpath(dirname(@__FILE__), ".."))
-const DEPS_DIR = joinpath(PKG_DIR, "deps")
-const ARCHIVE_DIR = joinpath(DEPS_DIR, "tzarchive")
-const TZ_SOURCE_DIR = joinpath(DEPS_DIR, "tzsource")
-const COMPILED_DIR = joinpath(DEPS_DIR, "compiled")
+PKG_DIR() = Pkg.dir("TimeZones")
```

Comment:
It is best to avoid using `Pkg.dir` as we may be working with a copy of TimeZones which isn't installed.